ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -93,6 +93,8 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    @objc public static let navigationBarBackgroundColor = UIColor(dynamicColor: FluentTheme.shared.aliasTokens.colors[.background3])
```

Comment:
We should not have a public static method that references the shared theme. Clients are expected to use their own theme information when accessing colors.

Reply:
Just to understand better, would you mind clarifying why? Also, would that be any different from creating a theme just to access this color in client code?

Reply:
Discussed offline, lemme know if you have any more questions!